src/gdcdictionary/schemas/pathology_detail.yaml

```diff
@@ -860,12 +982,47 @@ properties:
         - Base
         - Middle
 
+  tumor_measurement_method:
+    description: >-
+      The method used to measure tumor size.
+    enum:
+      - Echographic
+      - Pathologic
```

Comment:
do we need enumdef refs for these?

Reply:
bill let me know this depends on whether we can find a def in an external database